libbeat/_meta/config.reference.yml

```diff
@@ -256,7 +256,17 @@ output.elasticsearch:
   # The default is 50.
   #bulk_max_size: 50
 
-  # Configure http request timeout before failing an request to Elasticsearch.
+  # The number of seconds to wait before attempting to connect to Elasticsearch
+  # after an initial connection error. The default is 1s. If additional errors
+  # occur, the wait time is increased exponentially for each try until
+  # backoff.max is met.
+  #backoff.init: 1s
+
+  # The maximum number of seconds to wait before attempting to connect to
+  # Elasticsearch after a connection error. The default is 60s.
```

Comment:
Some more clarification:

`backoff.init` is applied on an network error. If connection was active, but connection gets lost or some other protocol/network error occurs, beats will disconnect and use `backoff.init`. After `backoff.init`, beats try to reconnect. If the reconnect fails, the backoff is increased exponentially up to `backoff.max`. After successful reconnect, the backoff timer is reset. On next network error, backoff will start with again with `backoff.init`.

Reply:
@dedemorton SGTM